### PR TITLE
chore(Bonus Pagamenti Digitali): [#176078094] Added supported credit cards page link

### PR DESCRIPTION
**Pull request by DavideValdo**
## List of changes proposed in this pull request
- Removed hardcoded supported card list
- Added link to external page

## How to test
Try to add a new card and see the link, this is the requested result from design:
download

The actual result is underlined being the default `<Link>` component

### CODE REVIEW

File: ts/screens/wallet/AddCardScreen.tsx [hunk 1]
```diff
@@ -166,6 +156,12 @@ class AddCardScreen extends React.Component<Props, State> {
     };
   }
 
+  private openSupportedCardsPage = (): void => {
+    Linking.openURL(acceptedCardsPageURL).catch(() =>
+      Alert.alert("", I18n.t("wallet.alert.supportedCardPageLinkError"))
```
Comment (debiff) [inline ts/screens/wallet/AddCardScreen.tsx:161]: This is a request of the design team? 
Maybe is better to show a toast:
`showToast(I18n.t("wallet.alert.supportedCardPageLinkError"))`

File: .env.local [hunk 2]
```diff
@@ -30,6 +30,8 @@ TOT_MESSAGE_FETCH_WORKERS=5
 SHUFFLE_PINPAD_ON_PAYMENT=NO
 # Repository of app content
 CONTENT_REPO_URL='http://127.0.0.1:3000/static_contents'
+# Accepted card info page URL 
+PAYMENT_METHODS_PAGE_URL='https://io.italia.it/metodi-pagamento'
```
Comment (debiff) [inline .env.local:34]: You should add this url also in the others `.env` files
- .env.production
- .env.local
- .env.example

**Review by debiff — CHANGES_REQUESTED** [review]

File: .env.local (hunk 2 above)
Comment (Undermaken) [reply .env.local:34]: we don't need to set this url as env variable.
It could be a const inside the code

**Review by debiff — APPROVED** [review]

File: ts/screens/wallet/AddCardScreen.tsx (hunk 1 above)
Comment (CrisTofani) [reply ts/screens/wallet/AddCardScreen.tsx:163]: as a safer implementation I suggest you to use `openWebUrl` function from https://github.com/pagopa/io-app/blob/3f4029c97879ab25ce3318c3d8cc34b95fe98ea9/ts%2Futils%2Furl.ts#L87

For the suggestion by @debiff maybe should be better to check it out with design team 😉

**Review by CrisTofani — CHANGES_REQUESTED** [review]

**Review by CrisTofani — APPROVED** [review]

**Pull request merged**